exporter/collector/metricsexporter.go

```diff
@@ -42,10 +42,17 @@ import (
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
+// self-observability reporting meters/tracers/loggers.
+type selfObservability struct {
```

Comment:
why define a new type if component.TelemetrySettings is already meant to encapsulate initialized telemetry instruments?